gridfinity-rebuilt-utility.scad

```diff
@@ -301,9 +335,9 @@ module cut_move_unsafe(x, y, w, h) {
     children();
 }
 
-module block_cutter(x,y,w,h,t,s) {
+module block_cutter(x,y,w,h,t,s,tab_width=d_tabw,tab_height=d_tabh) {
 
-    v_len_tab = d_tabh;
+    v_len_tab = tab_height;
     v_len_lip = d_wall2-d_wall+1.2;
     v_cut_tab = d_tabh - (2*r_f1)/tan(a_tab);
```

Comment:
This should be changed to
`v_cut_tab = tab_height - (2*r_f1)/tan(a_tab);`

A small line remained when using a smaller tab height. Now they look perfect, as far as i can see

Reply:
I would do it, but i don't know how after starting the pull request. Sorry!